language/documentation/book/src/generics.md

```diff
@@ -264,22 +264,21 @@ struct S3<phantom T> { f: S2<T> }
                              Error: Not a phantom position
 ```
 
-
 #### Instantiation
 
 When instantiating a struct,
 the arguments to phantom parameters are excluded when deriving the struct abilities.
 For example, consider the following code:
 
-```move=
+```move
 struct S<T1, phantom T2> has copy { f: T1 }
 struct NoCopy {}
 struct HasCopy has copy {}
 ```
 
 Consider now the type `S<HasCopy, NoCopy>`.
-Since `S` is defined with `copy` and all non-phantom arguments have copy
-then `S<HasCopy, NoCopy>` also has copy.
+Since `S` is defined with `copy` and all non-phantom arguments have copy ability
+then `S<HasCopy, NoCopy>` also has copy ability.
```

Comment:
I think the original phrasing is fine, unless there is something I'm missing. 

perhaps though this would help?
```suggestion
Since `S` is defined with `copy` and all non-phantom arguments have `copy`
then `S<HasCopy, NoCopy>` also has `copy`.
```